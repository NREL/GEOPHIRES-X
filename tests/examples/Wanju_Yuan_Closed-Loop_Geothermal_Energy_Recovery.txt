----------------------------------------------------------------------------------------
GEOPHIRES v3.0 Input File
Geothermal Electricity Example Problem using Percentage Thermal Drawdown Model (Example X) and closed loops
Oringinally created by NREL on 3/26/2018 as Example 4
Last modified on 2/26/2023

Example X Description: This example problem considers a simple reservoir 
at 2 km depth with an initial production temperature of 145deg.C. The thermal drawdown 
is assumed linear at 0.5%/year. The heat is converted to electricity with a subcritical
ORC. It is accessed by a closed loop well.
----------------------------------------------------------------------------------------
*** closed loop parameters ***
****************************************
Is AGS, True
Has Nonvertical Section, True
Multilaterals Cased, True
Plant Lifetime, 40, ---Years
Water Thermal Conductivity, 0.65
Total Nonvertical Length, 5001.0
Nonvertical Wellbore Diameter, 0.23495, -----m
Cylindrical Reservoir Radius of Effect Factor, 5.0
Closed Loop Calculation Start Year, 0.1
Number of Multilateral Sections, 3
Well Drilling Cost Correlation,3,				--- [-] Use built-in well drilling cost correlation #3 = vertical open-hole, large diameter
Horizontal Well Drilling Cost Correlation,1,				--- [-] Use built-in well drilling cost correlation #3 = vertical open-hole, large diameter
Reservoir Impedance, 1E-4, ----assume a very low reservoir impedance since the working fluid is never in contact with the reservoir
Production Flow Rate per Well, 1.666666,      -----kg/sec
Injection Temperature, 60,      -----C
Gradient 1,26.25,                          			---[deg.C/km]
Reservoir Depth, 4.0,      ----km
Cylindrical Reservoir Input Depth, 4.0, -----km
Cylindrical Reservoir Output Depth, 4.0, ------km
Cylindrical Reservoir Length, 5.0,     ----km
Reservoir Model,0,                        			---simople cylinder reservoir model
Number of Production Wells,1,            			---[-]
Number of Injection Wells,0,            			---[-]




*** Subsurface technical parameters ***
****************************************
Ramey Production Wellbore Model,0,	--- Should be 0 (disable) or 1 (enable)
Production Wellbore Temperature Drop,0,	--- [deg.C]
Production Flow Rate per Well,110, 	--- [kg/s]
Maximum Temperature,375,		--- [deg.C]
Reservoir Volume Option,4,		--- Should be 1 2 3 or 4. See manual for details.
Reservoir Volume,1e9,			--- [m3] (required for reservoir volume option 3 and 4
Reservoir Heat Capacity,1050,		--- [J/kg/K]


*** Surface technical parameters ***
************************************
End-Use Option,1,			--- [-] Electricity
Power Plant Type,1,			--- [1] Subcritical ORC
Circulation Pump Efficiency,0.8,	--- [-]
Plant Outlet Pressure, 68.95

*** Economic/Financial Parameters ***
*************************************
Economic Model,3,			--- Should be 1 (FCR model) 2 (Standard LCOE/LCOH model) or 3 (Bicycle model).
Fraction of Investment in Bonds,0.65,	--- [-] Required if Bicycle model is selected. See manual for details.
Inflated Bond Interest Rate,0.07,	--- [-] Required if Bicycle model is selected. See manual for details.
Inflated Equity Interest Rate,0.12,	--- [-] Required if Bicycle model is selected. See manual for details.
Inflation Rate,0.025,			--- [-] Required if Bicycle model is selected. See manual for details.
Combined Income Tax Rate,0.392,		--- [-] Required if Bicycle model is selected. See manual for details.
Gross Revenue Tax Rate,0,		--- [-] Required if Bicycle model is selected. See manual for details.
Reservoir Stimulation Capital Cost,0,				--- [M$/injection well] Reservoir stimulation capital cost per injection well


*** Simulation Parameters ***
Print Output to Console,1,		--- [-] Should be 0 (don't print results to console) or 1 (print results to console)
