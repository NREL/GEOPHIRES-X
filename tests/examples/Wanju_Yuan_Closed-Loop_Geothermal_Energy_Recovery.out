                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.4.0
 GEOPHIRES Build Date: 2022-06-30
 Simulation Date: 2024-02-13
 Simulation Time:  08:38
 Calculation Time:      1.616 sec

                           ***SUMMARY OF RESULTS***

      End-Use Option: Electricity
      Average Net Electricity Production:                     1.04 MW
      Electricity breakeven price:                           39.67 cents/kWh
      Number of production wells:                             1
      Number of injection wells:                              0
      Flowrate per production well:                         110.0 kg/sec
      Well depth (or total length, if not vertical):          9.0 kilometer
      Geothermal gradient:                                    0.0262 degC/m


                           ***ECONOMIC PARAMETERS***

      Economic Model  = BICYCLE
      Accrued financing during construction:                  0.00
      Project lifetime:                                      40 yr
      Capacity factor:                                       90.0 %
      Project NPV:                                         -37.66 MUSD
      Project IRR:                                           0.00 %
      Project VIR=PI=PIR:                                   -0.02
      Project MOIC:                                         -0.69

                          ***ENGINEERING PARAMETERS***

      Number of Production Wells:                             1
      Number of Injection Wells:                              0
      Well depth (or total length, if not vertical):          9.0 kilometer
      Water loss rate:                                        0.0
      Pump efficiency:                                       80.0
      Injection temperature:                                 60.0 degC
      User-provided production well temperature drop
      Constant production well temperature drop:              0.0 degC
      Flowrate per production well:                         110.0 kg/sec
      Injection well casing ID:                               8.000 in
      Production well casing ID:                              8.000 in
      Number of times redrilling:                             0
      Power plant type:                                       Subcritical ORC


                         ***RESOURCE CHARACTERISTICS***

      Maximum reservoir temperature:                        375.0 degC
      Number of segments:                                     1
      Geothermal gradient:                                    0.0262 degC/m


                           ***RESERVOIR PARAMETERS***

The AGS models contain an intrinsic reservoir model that doesn't expose values that can be used in extensive reporting.


                           ***RESERVOIR SIMULATION RESULTS***

      Maximum Production Temperature:                       120.0 degC
      Average Production Temperature:                       104.9 degC
      Minimum Production Temperature:                        99.7 degC
      Initial Production Temperature:                       120.0 degC
The AGS models contain an intrinsic reservoir model that doesn't expose values that can be used in extensive reporting.


                          ***CAPITAL COSTS (M$)***

         Drilling and completion costs:                      16.28 MUSD
         Drilling and completion costs per well:             16.28 MUSD
         Stimulation costs:                                   0.00 MUSD
         Surface power plant costs:                           6.72 MUSD
         Field gathering system costs:                        0.50 MUSD
         Total surface equipment costs:                       7.23 MUSD
         Exploration costs:                                  13.27 MUSD
      Total capital costs:                                   36.78 MUSD


                ***OPERATING AND MAINTENANCE COSTS (M$/yr)***

         Wellfield maintenance costs:                         0.23 MUSD/yr
         Power plant maintenance costs:                       0.30 MUSD/yr
         Water costs:                                         0.00 MUSD/yr
      Total operating and maintenance costs:                  0.53 MUSD/yr


                           ***SURFACE EQUIPMENT SIMULATION RESULTS***

      Initial geofluid availability:                          0.06 MW/(kg/s)
      Maximum Total Electricity Generation:                   1.66 MW
      Average Total Electricity Generation:                   1.04 MW
      Minimum Total Electricity Generation:                   0.86 MW
      Initial Total Electricity Generation:                   1.66 MW
      Maximum Net Electricity Generation:                     1.66 MW
      Average Net Electricity Generation:                     1.04 MW
      Minimum Net Electricity Generation:                     0.86 MW
      Initial Net Electricity Generation:                     1.66 MW
      Average Annual Total Electricity Generation:            8.12 GWh
      Average Annual Net Electricity Generation:              8.09 GWh
      Initial pumping power/net installed power:            100.00 %
      Average Pumping Power:                                  0.00 MW

                            ************************************************************
                            *  HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                            ************************************************************
  YEAR       THERMAL               GEOFLUID               PUMP               NET               FIRST LAW
             DRAWDOWN             TEMPERATURE             POWER             POWER              EFFICIENCY
                                     (degC)               (MW)              (MW)                  (%)
   1           1.0000                120.00               0.0027            1.6619                5.9339
   2           0.9166                109.99               0.0027            1.2250                5.2499
   3           0.9043                108.51               0.0027            1.1675                5.1558
   4           0.8972                107.67               0.0027            1.1354                5.1030
   5           0.8923                107.08               0.0027            1.1134                5.0665
   6           0.8886                106.63               0.0027            1.0968                5.0389
   7           0.8856                106.27               0.0027            1.0834                5.0167
   8           0.8830                105.96               0.0027            1.0723                4.9982
   9           0.8808                105.70               0.0027            1.0628                4.9824
  10           0.8789                105.47               0.0027            1.0546                4.9686
  11           0.8772                105.27               0.0027            1.0473                4.9563
  12           0.8757                105.08               0.0027            1.0408                4.9454
  13           0.8743                104.92               0.0027            1.0348                4.9355
  14           0.8731                104.77               0.0027            1.0294                4.9264
  15           0.8719                104.63               0.0027            1.0245                4.9181
  16           0.8708                104.50               0.0027            1.0199                4.9104
  17           0.8698                104.37               0.0027            1.0156                4.9033
  18           0.8688                104.26               0.0027            1.0117                4.8966
  19           0.8680                104.16               0.0027            1.0079                4.8903
  20           0.8671                104.05               0.0027            1.0044                4.8844
  21           0.8663                103.96               0.0027            1.0011                4.8789
  22           0.8656                103.87               0.0027            0.9980                4.8736
  23           0.8649                103.78               0.0027            0.9950                4.8686
  24           0.8642                103.70               0.0027            0.9922                4.8639
  25           0.8635                103.63               0.0027            0.9895                4.8593
  26           0.8629                103.55               0.0027            0.9870                4.8550
  27           0.8623                103.48               0.0027            0.9845                4.8509
  28           0.8618                103.41               0.0027            0.9822                4.8469
  29           0.8612                103.35               0.0027            0.9799                4.8431
  30           0.8607                103.28               0.0027            0.9777                4.8394
  31           0.8602                103.22               0.0027            0.9756                4.8359
  32           0.8597                103.16               0.0027            0.9736                4.8325
  33           0.8592                103.10               0.0027            0.9717                4.8292
  34           0.8587                103.05               0.0027            0.9698                4.8260
  35           0.8583                103.00               0.0027            0.9680                4.8229
  36           0.8579                102.94               0.0027            0.9662                4.8199
  37           0.8575                102.89               0.0027            0.9645                4.8171
  38           0.8570                102.85               0.0027            0.9628                4.8143
  39           0.8567                102.80               0.0027            0.9612                4.8115
  40           0.8563                102.75               0.0027            0.9597                4.8089


                              *******************************************************************
                              *  ANNUAL HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                              *******************************************************************
  YEAR             ELECTRICITY                   HEAT                RESERVOIR            PERCENTAGE OF
                    PROVIDED                   EXTRACTED            HEAT CONTENT        TOTAL HEAT MINED
                   (GWh/year)                  (GWh/year)            (10^15 J)                 (%)
   1                  11.9                       208.1                 32.76                   2.24
   2                   9.0                       175.5                 32.13                   4.12
   3                   9.4                       180.3                 31.48                   6.06
   4                   8.7                       171.8                 30.86                   7.90
   5                   8.9                       174.3                 30.23                   9.78
   6                   8.5                       169.4                 29.62                  11.60
   7                   8.6                       171.0                 29.00                  13.43
   8                   8.3                       167.6                 28.40                  15.23
   9                   8.4                       168.7                 27.79                  17.05
  10                   8.2                       166.2                 27.20                  18.83
  11                   8.3                       166.9                 26.60                  20.63
  12                   8.1                       165.0                 26.00                  22.40
  13                   8.2                       165.5                 25.41                  24.18
  14                   8.1                       164.1                 24.81                  25.94
  15                   8.1                       164.3                 24.22                  27.71
  16                   8.0                       163.3                 23.64                  29.46
  17                   8.0                       163.3                 23.05                  31.21
  18                   8.0                       162.6                 22.46                  32.96
  19                   7.9                       162.4                 21.88                  34.71
  20                   7.9                       161.9                 21.29                  36.45
  21                   7.9                       161.6                 20.71                  38.18
  22                   7.9                       161.4                 20.13                  39.92
  23                   7.8                       160.8                 19.55                  41.64
  24                   7.8                       160.9                 18.97                  43.37
  25                   7.8                       160.1                 18.40                  45.09
  26                   7.8                       160.5                 17.82                  46.82
  27                   7.7                       159.5                 17.25                  48.53
  28                   7.8                       160.1                 16.67                  50.25
  29                   7.7                       158.8                 16.10                  51.96
  30                   7.8                       159.9                 15.52                  53.67
  31                   7.6                       158.1                 14.95                  55.37
  32                   7.7                       159.7                 14.38                  57.09
  33                   7.6                       157.4                 13.81                  58.78
  34                   7.7                       159.7                 13.24                  60.50
  35                   7.5                       156.5                 12.67                  62.18
  36                   7.8                       160.0                 12.10                  63.90
  37                   7.4                       155.0                 11.54                  65.56
  38                   7.9                       161.5                 10.96                  67.30
  39                   7.1                       150.5                 10.42                  68.92
  40                   7.3                       138.0                  9.92                  70.40
